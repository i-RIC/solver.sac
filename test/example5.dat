T1   EXAMPLE1:  Simple reach data
SI        3 
XS   SEC3 304.8
GR        -36.88,30.114 -32.31,29.900 -27.43,28.773 -25.91,27.706 
GR        -24.38,27.036 -22.86,27.036 -21.34,27.249 -19.81,27.645
GR        -18.29,27.706 -16.46,27.798 -15.54,29.108 0.,29.627
GR        20.42,29.748 25.30,30.084
N         0.035 0.040 0.030
SA        -32.31, -15.54 
HP 4 SEC3 30.099
XS   SEC2 420.62
GR        0.61,30.175 4.57,29.779 8.23,29.748 10.06,27.737 11.58,27.371 
GR        13.101,27.371 14.63,27.493 16.15,27.554 17.68,27.584
GR        18.29,27.767 20.73,29.200 23.77,29.809 56.39,30.236
GR        71.32,30.297 
N         0.030 0.037 0.030
SA        8.23,23.77
HP 4 SEC2 30.236
XS   SEC1 507.80
GR        3.69,30.358 4.88,30.145 7.92,30.022 9.14,28.346 9.754,27.584
GR        11.28,27.798 12.80,27.706 14.33,27.920 15.85,28.103 17.37,28.224
GR        18.90,28.407 20.42,27.950 21.95,28.346 24.99,29.992 
GR        77.72,30.175 80.80,30.450
N         0.030 0.045 0.030
SA        7.92,24.99 
HP 4 SEC1 30.404
