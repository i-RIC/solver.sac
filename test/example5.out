SAC -USGS slope-area program Ver 2.0                                    page  0

Echo input data file 
Simple reach example with metric input and feet-second output                   
XS   SEC3 304.8                                                                 
GR        -36.88,30.114 -32.31,29.900 -27.43,28.773 -25.91,27.706               
GR        -24.38,27.036 -22.86,27.036 -21.34,27.249 -19.81,27.645               
GR        -18.29,27.706 -16.46,27.798 -15.54,29.108 0.,29.627                   
GR        20.42,29.748 25.30,30.084                                             
N         0.035 0.040 0.030                                                     
SA        -32.31, -15.54                                                        
HP   SEC3 30.099                                                                
XS   SEC2 420.62                                                                
GR        0.61,30.175 4.57,29.779 8.23,29.748 10.06,27.737 11.58,27.371         
GR        13.101,27.371 14.63,27.493 16.15,27.554 17.68,27.584                  
GR        18.29,27.767 20.73,29.200 23.77,29.809 56.39,30.236                   
GR        71.32,30.297                                                          
N         0.030 0.037 0.030                                                     
SA        8.23,23.77                                                            
HP   SEC2 30.236                                                                
XS   SEC1 507.80                                                                
GR        3.69,30.358 4.88,30.145 7.92,30.022 9.14,28.346 9.754,27.584          
GR        11.28,27.798 12.80,27.706 14.33,27.920 15.85,28.103 17.37,28.224      
GR        18.90,28.407 20.42,27.950 21.95,28.346 24.99,29.992                   
GR        77.72,30.175 80.80,30.450                                             
N         0.030 0.045 0.030                                                     
SA        7.92,24.99                                                            
HP   SEC1 30.404                                                                
SAC -USGS slope-area program Ver 2.0                                    page  1

Simple reach example with metric input and feet-second output                   
EXAMPLE1:  Simple reach data                                               
                                                                           
                                                                           

                             DISCHARGE COMPUTATIONS
                      Reach
             dH,fall  length  Discharge  Spread    HF     CX     RC     RX  ER
               (ft)     (ft)     (cfs)     (%)    (ft)
 SEC1 - SEC2   0.55     286.      1924.      0   0.392  1.000  0.404  0.000     
 SEC2 - SEC3   0.45     380.      2108.     26   0.555  0.876  0.000 -0.379 *   

 SEC1 - SEC3   1.00     666.      2001.      9   0.923  0.952  0.185 -0.205     

Definitions:
  Spread, the percent difference between discharge computed with no expansion
      loss (k=0) and discharge computed with full expansion loss (k=1.0), divided
      by the discharge computed with full expansion loss
  HF, friction head- HF = sum of Q*Q*L/(K1*K2) over subreaches; Q, discharge;
      L, reach length; K1, upstream section conveyance;
      K2, downstream section conveyance
  CX, the computed discharge divided by the discharge computed with no expansion
      loss (k=0)
  RC, velocity head change in contracting section divided by friction head
  RX, velocity head change in expanding section divided by friction head
  ER, warnings, *-fall <' 0.5ft, @-conveyance ratio exceeded, #-reach too short
      error, 1-negative or 0 fall
  ******, terms that can not be computed because' of strong expansion in reach

                              CROSS  SECTION  PROPERTIES
--------------------------------------------------------------------------------
  I.D. SEC3                   Velocity head    0.22ft    Discharge     2001.cfs
   Ref.distance     1000.ft             Q/K    0.0011        Alpha 1.200  

  Sub  Water                  Top   Wetted   Hydraulic   Conveyance
 area surface   n     Area   width perimeter   radius   x 0.001      Vel.    F
  no. el.(ft)       (sq.ft)   (ft)   (ft)       (ft)     (cfs)   %   (fps)
    1   98.75 0.035      4.6    13.9   14.0      0.33      0.092   0.  0.7  0.21
    2   98.75 0.040    359.7    55.0   59.5      6.05     44.491  74.  4.1  0.28
    3   98.75 0.030    222.4   134.0  134.1      1.66     15.478  26.  2.3  0.32
Total   98.75   ---     587.    203.   208.      2.83     60.061 100.  3.4  0.35
--------------------------------------------------------------------------------
Definitions:
  n, Manning's coefficient of roughness    Q/K = discharge/conveyance
  F, Froude number F = Ki*Q/(K*A sqrt(g*(Ai/TWi)); Q, discharge; A, total cross-
      section area; g, acceleration of gravity; Ai, sub-section area; TWi, sub-
      section top width
SAC -USGS slope-area program Ver 2.0                                    page  2

Simple reach example with metric input and feet-second output                   
EXAMPLE1:  Simple reach data                                               
                                                                           
                                                                           

                              CROSS  SECTION  PROPERTIES
--------------------------------------------------------------------------------
  I.D. SEC2                   Velocity head    0.41ft    Discharge     2001.cfs
   Ref.distance     1380.ft             Q/K    0.0016        Alpha 1.299  

  Sub  Water                  Top   Wetted   Hydraulic   Conveyance
 area surface   n     Area   width perimeter   radius   x 0.001      Vel.    F
  no. el.(ft)       (sq.ft)   (ft)   (ft)       (ft)     (cfs)   %   (fps)
    1   99.20 0.030     29.7    25.0   25.3      1.17      1.639   3.  2.2  0.35
    2   99.20 0.037    341.4    51.0   55.6      6.14     46.092  91.  5.3  0.36
    3   99.20 0.030     75.0   107.0  107.0      0.70      2.936   6.  1.5  0.33
Total   99.20   ---     446.    183.   188.      2.37     50.667 100.  4.5  0.51
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
  I.D. SEC1                   Velocity head    0.23ft    Discharge     2001.cfs
   Ref.distance     1666.ft             Q/K    0.0014        Alpha 1.290  

  Sub  Water                  Top   Wetted   Hydraulic   Conveyance
 area surface   n     Area   width perimeter   radius   x 0.001      Vel.    F
  no. el.(ft)       (sq.ft)   (ft)   (ft)       (ft)     (cfs)   %   (fps)
    1   99.75 0.030     12.4    13.9   14.1      0.88      0.568   1.  1.7  0.32
    2   99.75 0.045    386.9    56.0   61.9      6.25     43.454  81.  4.2  0.28
    3   99.75 0.030    185.1   181.4  181.4      1.02      9.314  17.  1.9  0.33
Total   99.75   ---     584.    251.   257.      2.27     53.336 100.  3.4  0.40
--------------------------------------------------------------------------------
Definitions:
  n, Manning's coefficient of roughness    Q/K = discharge/conveyance
  F, Froude number F = Ki*Q/(K*A sqrt(g*(Ai/TWi)); Q, discharge; A, total cross-
      section area; g, acceleration of gravity; Ai, sub-section area; TWi, sub-
      section top width
